File: capivaraprojects/greeneyes/models/Image.py
from capivaraprojects.greeneyes.models.Disease import Disease
from capivaraprojects.greeneyes.models.Type import Type
class Image:
    def __init__(self, 
            id=0, 
            disease=Disease(), 
            url="", 
            description="", 
            source="", 
            size = Type()):
        self.id = id
        self.disease = disease
        self.url = url
        self.description = description
        self.source = source

File: capivaraprojects/greeneyes/models/Image.js
const Disease = require('./Disease.js');
const Type = require('./Type.js');

class Image {
	constructor (id=0, disease = Disease(), url = "", description = "", source = "", size = Type()) {
		this.id = id;
		this.disease = disease;
		this.url = url;
		this.description = description;
		this.source = source;
		this.size = size;
	}
}